apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/grpc/GrpcClients.java

```diff
@@ -47,6 +47,14 @@ public GrpcClients(@GrpcClient("bridge") StargateBridge bridge, GrpcMetadataConf
     this.bridge = bridge;
   }
 
+  public Metadata.Key<String> getTenantIdKey() {
```

Comment:
pritty sure don't need this class anymore, you can inject `GrpcMetadataConfig config` in the interceptor directly